.github/workflows/integration-opendistro

```diff
@@ -0,0 +1,19 @@
+name: Integration tests with OpenDistro
+
+on: [push, pull_request]
+
+jobs:
+  integration:
+    name: Integ
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        cluster: ["opendistro"]
```

Comment:
you don't need this as a parameter. You can directly use opendistro at line 19

Reply:
My bad 🤦🏼‍♀️